src/Faithlife.Utility/ArrayUtility.cs

```diff
@@ -32,9 +32,9 @@ public static class ArrayUtility
 		/// item of the first array is equal to the first item of the second array, etc.</returns>
 		public static bool AreEqual<T>(T[] array1, T[] array2, IEqualityComparer<T> comparer)
 		{
-			if (array1 == null)
-				return array2 == null;
-			if (array2 == null)
+			if (array1 is null)
```

Comment:
I know our team has had some questions on if using `object is null` is preferred over `object == null` in general usage. I prefer `is null` since null is what I am checking for and I don't really care if a class overrides `Equals``. What are your thoughts on this?

Reply:
I agree with Jared Parsons (on the C# compiler team) that C# developers should embrace `is null` and `is object` as the canonical (non) nullability tests in modern C#.

https://twitter.com/jaredpar/status/1171478965355024384

`x is null` should be preferred over `x == null` in all situations: it always emits efficient IL and if `x` can't be `null` (e.g., it's a value type) you'll get a compiler error (but `!= null` will compile).